dbms/src/AggregateFunctions/AggregateFunctionNull.h

```diff
@@ -429,12 +429,19 @@ class AggregateFunctionNullUnary final : public AggregateFunctionNullBase<result
         if constexpr (input_is_nullable)
         {
             const ColumnNullable * column = static_cast<const ColumnNullable *>(columns[0]);
+            const String & nexted_func_name = this->nested_function->getName();
             if (!column->isNullAt(row_num))
             {
                 this->setFlag(place);
                 const IColumn * nested_column = &column->getNestedColumn();
                 this->nested_function->add(this->nestedPlace(place), &nested_column, row_num, arena);
             }
+            else if (nexted_func_name == "groupBitOr" || nexted_func_name == "groupBitAnd" || nexted_func_name == "groupBitXor")
```

Comment:
```suggestion
            if (!column->isNullAt(row_num))
            {
                this->setFlag(place);
                const IColumn * nested_column = &column->getNestedColumn();
                this->nested_function->add(this->nestedPlace(place), &nested_column, row_num, arena);
            }
            else if (const String & nexted_func_name = this->nested_function->getName(); nexted_func_name == "groupBitOr" || nexted_func_name == "groupBitAnd" || nexted_func_name == "groupBitXor")
```
in this way looks better?

Reply:
Fine.